meta/saiserialize.cpp

```diff
@@ -2968,3 +2968,21 @@ void sai_deserialize_queue_attr(
 
     sai_deserialize_enum(s, &sai_metadata_enum_sai_queue_attr_t, (int32_t&)attr);
 }
+
+void sai_deserialize_media_change_ntf(
```

Comment:
which attribute? do we already have the serialize for the vector<uint32_t>?

Reply:
Hi Guohan
The data is already serialized in media_settings.json in json format. Now the task is to de-serialize the data from the json format to the port attributes(SAI_PORT_ATTR_SERDES_PREEMPHASIS, SAI_PORT_ATTR_SERDES_IDRIVER, SAI_PORT_ATTR_SERDES_IPREDRIVER)

Reply:
I do not see how this SAI serialize is related to media_settings.json. they are parsed by xrcvd, send to orchagent app db, then go to sai.